hal/src/electron/modem/enums_hal.h

```diff
@@ -112,6 +112,24 @@ typedef struct {
     unsigned int ci;     //!< Cell ID in hexadecimal format (2 to 4 bytes in hex)
 } NetStatus;
 
+#ifdef __cplusplus
+#define MDM_R410_EDRX_ACTS_MAX (4) //!< maximum number of AcTs for eDRX mode on SARA-R410M-02B
+//! eDRX AcTs
+struct MDM_EdrxActs {
+    uint16_t size;
```

Comment:
This structure is only being used internally, there's no need for it to support versioning

Reply:
I'll reduce it's size, and move it to mdm_hal.h as well.